docs/marimo/mlir_ir.py

```diff
@@ -103,7 +107,12 @@ def _(first_text, mo):
 
 @app.cell(hide_code=True)
 def _(exercise_text, first_text_area, mo):
-    _first_info_text = exercise_text(first_text_area.value, "first", ((1, 2), (3, 4)), ("first(1, 2) = ", "first(3, 4) = "))
+    _first_info_text = exercise_text(
+        first_text_area.value,
+        "first",
+        ((1, 2), (3, 4)),
+        ("first(1, 2) = ", "first(3, 4) = "),
+    )
```

Comment:
we have ruff formatting exclude notebooks, so it's my local settings I assum.

Reply:
Are you editing the raw python locally or using marimo edit?

Reply:
Raw python locally, do you want me to run with the editor? I'll see if it can reformat these.

Reply:
Yeah it would be good to run it in the editor, before merging because they have their own formatter that will likely conflict with your changes.

Reply:
Yeah, there is no "format all cells", I have to go over each cell and do a `ctrl-b`

Reply:
"Run all cells" seems to do it